patterns/interstitial-with-background.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
Let's try to move this to a group style and apply the background that way. We want to try to greater one pattern to rule them all here. and having the group class added (see above) will help achieve this. It is totally fine to have custom styles added to this to lock it down.